util/design/lib/OtpMemMap.py

```diff
@@ -59,18 +62,35 @@ def _validate_scrambling(scr):
         random_or_hexvalue(dig, "cnst_value", scr["cnst_size"] * 8)
 
 
+def _calc_part_size(part):
+    size = 0
+    print ("working on partition {}".format(part['name']))
+    for item in part['items']:
+        size += int(item['size'])
+
+    size = SCRAMBLE_BLOCK_WIDTH * \
+        int((size + SCRAMBLE_BLOCK_WIDTH - 1) / SCRAMBLE_BLOCK_WIDTH)
+
+    # Add on digest portion
+    if part['hw_digest']:
```

Comment:
I think this applies in the `sw_digest` case as well -> `if part["sw_digest"] or part["hw_digest"]:`

Reply:
o yeah good point.

Reply:
yeah, the digest size is currently fixed to one scrambling block for simplicity (other parts of the script assume this).